README.md

```diff
@@ -75,4 +75,126 @@ $ pytest .\tests_e2e\
 
 #### Statuses
 
-This application relies on the existence of Lists on the Trello board named "`To Do`", "`Doing`", and "`Done`" (the default set). The code is extendable to use further lists by calling `statuses` directly from the `ViewModel` rather than the `board_statuses` subset (the default set) returned by `getBoardStatuses`. "`Done`" is still required to facilitate the `/complete/<id>` route. An alternative solution for this would be to simply set the "complete" item with the _last_ status in the list of statuses returned from Trello, e.g. `item['status'] = getStatuses()[-1].title`. Another solution would be to remove the route as the functionality to mark a task as "complete" is fulfilled by the drag and drop feature.
\ No newline at end of file
+This application relies on the existence of Lists on the Trello board named "`To Do`", "`Doing`", and "`Done`" (the default set). The code is extendable to use further lists by calling `statuses` directly from the `ViewModel` rather than the `board_statuses` subset (the default set) returned by `getBoardStatuses`. "`Done`" is still required to facilitate the `/complete/<id>` route. An alternative solution for this would be to simply set the "complete" item with the _last_ status in the list of statuses returned from Trello, e.g. `item['status'] = getStatuses()[-1].title`. Another solution would be to remove the route as the functionality to mark a task as "complete" is fulfilled by the drag and drop feature.
+
+### Virtual environments
+
+This application can also be run within a virtual machine (VM) by using [Vagrant](https://www.vagrantup.com/downloads.html).
+
+Vagrant requires a hypervisor installed. This application is configured to use Windows [Hyper-V](https://docs.microsoft.com/en-us/virtualization/hyper-v-on-windows/about/). 
```

Comment:
It is not configured to use HyperV. The Vagrantfile doesn't require a specific provider. See my comment in the Vagrantfile for details.

Reply:
Acknowledged. Made changes to wording in md.